linux/utils/check_guest_os_gui.yml

```diff
@@ -1,6 +1,18 @@
 # Copyright 2021-2022 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
+# Set default guest_os_with_gui by checking Xorg exists or not
```

Comment:
Can this part be put to the end when "guest_os_with_gui" is not defined then execute this part to try to get it?